internal/runtime/cpu_quota_linux.go

```diff
@@ -44,6 +45,9 @@ func CPUQuotaToGOMAXPROCS(minValue int) (int, CPUQuotaStatus, error) {
 	}
 
 	maxProcs := int(math.Floor(quota))
+	if roundOpt == Ceil {
+		maxProcs = int(math.Ceil(quota))
+	}
```

Comment:
readability suggestion: this might be an if-else situation over `x := default; if { x = override }` since both involve computation.

```suggestion
	var maxProcs int
	if roundOpt == Ceil {
		maxProcs = int(math.Ceil(quota))
	} else {
		maxProcs = int(math.Floor(quota))
	}
```